base_tier_validation/models/tier_review.py

```diff
@@ -153,10 +153,42 @@ def _get_reviewers(self):
         reviewer_field = self.env["res.users"]
         if self.reviewer_field_id:
             resource = self.env[self.model].browse(self.res_id)
-            reviewer_field = getattr(resource, self.reviewer_field_id.name, False)
-            if not reviewer_field or not reviewer_field._name == "res.users":
+            field_name = self.reviewer_field_id.name
+            no_field_on_model = field_name not in resource._fields
+            if no_field_on_model:
+                msg = self.env._(
+                    "There are no field %(field_name)s on the %(model_name)s model",
+                )
+                raise ValidationError(
+                    msg % {"field_name": field_name, "model_name": resource._name}
+                )
```

Comment:
```suggestion
                raise ValidationError(
                    self.env._(
                        "There are no field %(field_name)s on the %(model_name)s model",
                        field_name=field_name,
                        model_name=resource._name,
                    )
                )
```

use Odoo's gettext built-in string interpolation
This lets Odoo recover gracefully from incorrect translation terms



(same applies to other cases below)